schemahcl/hcl_test.go

```diff
@@ -361,83 +361,6 @@ func TestMultiFile(t *testing.T) {
 	}, test.People[1])
 }
 
-func TestDynamicBlocks(t *testing.T) {
```

Comment:
is this added elsewhere?

Reply:
No. I decided to remove it as it's not needed anymore, and it just adds complexity to the current package.